ensime-company.el

```diff
@@ -125,7 +125,7 @@
   (yas-minor-mode-on)
   (set (make-local-variable 'company-idle-delay) 0)
   (set (make-local-variable 'company-minimum-prefix-length) 2)
-  (local-set-key [tab] 'ensime-company-complete-or-indent))
+  (local-set-key (kbd "TAB") 'ensime-company-complete-or-indent))
```

Comment:
I use a combination of `[tab]` and `"<tab>"` in my `.emacs`. Can you please point to some docs that describe the different? I don't want to break it for everybody in the various Windowed OSes.